client/package.json

```diff
@@ -23,5 +23,6 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "proxy": "http://193.1.131.26:8888/"
```

Comment:
this is fine for now, we should look for generalisation and not hard code any values for that matter